OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
Also I can augment #1783 so that `replaceMarkerSet` no longer takes a `State`. That would further improve the consistency of the two methods.